pontoon/base/models/translated_resource.py

```diff
@@ -1,149 +1,79 @@
 import logging
 
-from typing import Any
-
 from django.db import models
-from django.db.models import Q, Sum
+from django.db.models import F, Q, Sum
 
-from pontoon.base import utils
-from pontoon.base.models.aggregated_stats import AggregatedStats
 from pontoon.base.models.entity import Entity
 from pontoon.base.models.locale import Locale
 from pontoon.base.models.project import Project
-from pontoon.base.models.project_locale import ProjectLocale
 from pontoon.base.models.resource import Resource
 from pontoon.base.models.translation import Translation
+from pontoon.base.models.user import User
 
 
 log = logging.getLogger(__name__)
 
 
 class TranslatedResourceQuerySet(models.QuerySet):
-    def aggregated_stats(self):
-        return self.aggregate(
-            total=Sum("total_strings"),
-            approved=Sum("approved_strings"),
-            pretranslated=Sum("pretranslated_strings"),
-            errors=Sum("strings_with_errors"),
-            warnings=Sum("strings_with_warnings"),
-            unreviewed=Sum("unreviewed_strings"),
+    def string_stats(
+        self, user: User | None = None, *, show_disabled: bool = False
+    ) -> dict[str, int]:
+        query = (
+            self
+            if show_disabled
+            else self.filter(
+                resource__project__disabled=False,
+                resource__project__system_project=False,
```

Comment:
I find the name `show_disabled`, given it also impacts `system_projects`.

 ```suggestion
             if show_localizable
             else self.filter(
                 resource__project__disabled=False,
                 resource__project__system_project=False,
```

Reply:
Maybe we should _always_ filter out disabled, and have a toggle for system projects?